example/profile.php

```diff
@@ -1,11 +1,11 @@
 <?php
-require_once './vendor/autoload.php';//make sure you run composer update inside the example folder before trying them
+require_once './vendor/autoload.php'; // Make sure you run composer update inside the example folder before trying them
 $errorMsg = '';
-//get and sanitize the token
+// Get and sanitize the token
 $token = (isset($_GET['token']))? preg_replace('/[^-a-zA-Z0-9_]/', '', $_GET['token']) : '';
 $config = [
-    'sdkId' => 'add your SDK ID here', //this is your SDK ID associated with the Yoti Application you created on Dashboard
-    'pemFile' => __DIR__ . '/keys/your-key-name.pem', //This is the public key (in .pem format) associated with the Yoti Application you created on Dashboard
+    'sdkId' => 'add your SDK ID here', // This is your SDK ID associated with the Yoti Application you created on Dashboard
```

Comment:
Not sure these comments are necessary, as we moved the details into the Readme file. If you decide to keep them, `in the Yoti Dashboard` sounds better to me.